docs/blog/2018-2-6-choosing-a-back-end/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Picking a back-end for GatsbyJS
+date: "2018-02-06"
+author: "Ross Whitehouse"
+---
+
+# GatsbyJS: Picking a back-end
+A little while ago I had the six-monthly itch to redo my portfolio site, and I decided I was going to learn to use Gatsby. But that seems to be only half the battle. After you’ve built your front-end, how do you decide on the back-end? There’s so many out there!
+
+![GatsbyJS](gatsby.jpeg)
+
+## Background - Why Gatsby?
+The only thing as abundant than Headless CMS options is Static Site Generator options. From Hugo (built on Google’s Go language) to Jekyll (Ruby) and even Nuxt which is based on VueJS. I picked Gatsby out of the pack for a number of reasons, chief amongst which is that I’m professionally a React.js developer, so getting to use React in these quick projects is a great benefit.
+
+Gatsby pitches as a “Blazing-fast static site generator for React”. So if I can build sites, but also encompass some React knowledge, all the better right? And I’m always looking for side-projects and sites for others, so if I can get a JAMstack sorted quicker and easier than my previous WordPress stack, then all the better going forward! I thought it was best to start with my own site, to test the viability.
+
+> __What-stack?__ JAM stands for Javascript, API, Markup. There are lots of benefits to this type of site, but the one I’m interested in is that your CMS is detached from your site - no hefty WordPress build for your tiny blog site.
```

Comment:
Do you want this to appear like a quote? If so, it has to be with the arrows at the beginning of each line in the whole block that you want to appear like a quote. I'll try to do an example here with extra spaces after the arrows so it doesn't automatically format it

>  line of quote
>  line of quote
>  line of quote

Reply:
Line 17 starting '__What-stack' is a quote, but line 15 isn't. The quoted lines should be 17, 18, 22, 23, 34, 60, 96

Reply:
Ah yes, my comment was just about line 17 because it is so long. All I know is that by looking at other quotes in other blog posts, most people will break their quote into shorter segments and put a > at the beginning of each new line instead of leaving it as one long line. That way you get to choose where the line breaks are. Is this right, @KyleAMathews?